data/campaigns/Heir_To_The_Throne/achievements.cfg

```diff
@@ -1 +1,552 @@
 #textdomain wesnoth-httt
+
+[achievement_group]
+    display_name=_"Heir to the Throne"
+    content_for=Heir_To_The_Throne_achievements
+
+    [achievement]
+        id="HttT_S1_Defend_Aethenwood"
+        name=_"Defend Aethenwood from the Orcs"
+        name_completed= _ "Defended Aethenwood from the Orcs"
+        description=_"Defeat any two of the three enemy orcish commanders."
+        description_completed=_"Slain two of the three enemy orcish commanders."
```

Comment:
"Slain" doesn't sound quite right to me... but then again, I'm not sure if "Slayed" or "Slew" sound much better...

Reply:
"slain" is the past participle, "slew" is the past tense. So "slew" is probably the correct choice here.